Food_Tracker_App/app/src/main/res/layout/meal_plan_scale_ingredient_dialog.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="16dp">
+
+
+    <ImageButton
+        android:id="@+id/closeButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="right"
+        android:background="@android:color/transparent"
+        app:srcCompat="@drawable/ic_close_fill0_wght400_grad0_opsz48" />
+
+    <ImageView
+        android:id="@+id/imageView2"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="88dp"
+        android:layout_marginRight="88dp"
+        android:adjustViewBounds="true"
+        android:scaleType="centerCrop"
+        android:layout_centerInParent="true"
+        app:srcCompat="@drawable/ic_egg_alt_fill0_wght400_grad0_opsz48_black" />
```

Comment:
Here just hardcode a layout height and a layout width, on a big screen this would be ENORMOUS 😆, say `96dp` or slightly bigger if you want, and remove margins and use `layout_gravity="center"`